egs/lre07/v2/local/callfriend_lang.txt

```diff
@@ -0,0 +1,7 @@
+48 french
+49 arabic.standard
+54 korean
+55 chinese.mandarin.mainland
+56 chinese.mandarin.taiwan
+57 spanish.caribbean
+58 spanish.noncaribbean
```

Comment:
I don't see this file referenced anywhere. _If_ it isn't needed, can we remove it?